src/imeta.cpp

```diff
@@ -2208,6 +2227,10 @@ void usageMain() {
         "The appropriate zone (local or remote) is determined from the path names",
         "or via -z Zonename (for 'qu' and when working with resources).",
         " ",
+        "If you are the iRODS administrator, you can include the -M option to",
```

Comment:
"the iRODS administrator" -> "an iRODS administrator" ?

Reply:
good catch. will change it.

Reply:
done